windowsforms/FlexGridShowcaseDemo/cs/MainForm.cs

```diff
@@ -0,0 +1,682 @@
+﻿using C1.Win.FlexGrid;
+using C1.Win.Themes;
+using C1.Win.Ribbon;
+using C1.Win.RulesManager;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Forms;
+
+namespace FlexGridShowcaseDemo
+{
+
+    public partial class MainForm : C1RibbonForm 
+    {
+        private const string _customThemeName = "Greenwich";
+        private const int _footerTextPadding = 4;
+        private DataSet _dataSet = new DataSet();
+        private C1RulesManager _rulesManager = new C1RulesManager();
+        private IEnumerable<IRule> _rules = Enumerable.Empty<IRule>();
+        private static Random _rnd = new Random();
+
+        #region Initialization
+
+        public MainForm()
+        {
+            InitializeComponent();
+        }
+
+        private void MainForm_Load(object sender, EventArgs e)
+        {
+            FillData();
+
+            InitImages();
+            InitFlexGrid();
+            InitRules();
+            InitThemes();
+
+            // Data sizes
+            var dataSizes = new List<string>() { "10 Rows, 12 Columns", "50 Rows, 12 Columns", "100 Rows, 12 Columns", "1000 Rows, 12 Columns", "5000 Rows, 12 Columns" };
```

Comment:
I think that's more readable.

```suggestion
            var dataSizes = new List<string>()
            {
                "10 Rows, 12 Columns",
                "50 Rows, 12 Columns",
                "100 Rows, 12 Columns",
                "1000 Rows, 12 Columns",
                "5000 Rows, 12 Columns"
            };
```

Reply:
Fixed with 90b88b70de72609553a146f7a8ef39b3afb8ed61.